notebooks/official/pytorch/pytorch-text-sentiment-classification-custom-train-deploy.ipynb

```diff
@@ -0,0 +1,1979 @@
+{
```

Comment:
<p>hyper parameters -&gt; hyperparameters</p><p> </p><p>Also... -&gt; It also...</p><p> </p><p>Includes function -&gt; Includes a function</p><p> </p><p>create model -&gt; create a model</p><p> </p><p>Defines metadata -&gt; Defines the metadata</p><p> </p><p>classification task -&gt; classification tasks,</p><p> </p><p>, target labels. -&gt; and target labels.</p><p> </p><p>Includes utility functions, such as those used for reading data and saving models to Cloud Storage buckets.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
updated.